pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -32,11 +33,13 @@ import (
 	rmeta "github.com/tigera/operator/pkg/render/common/meta"
 	rsecret "github.com/tigera/operator/pkg/render/common/secret"
 	"github.com/tigera/operator/pkg/render/logstorage/esgateway"
+
```

Comment:
If you want to adjust these imports to ourt standard format, please have 3 sections:
1. golang packages (this group is already there)
2. non-tigera packages
3. tigera packages
With a newline between each grouping.